js/Marquee.js

```diff
@@ -0,0 +1,30 @@
+let url =
+  "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/stock-screener?exchange=NASDAQ&limit=150 ";
```

Comment:
Instead of rewriting the API URL ("https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/) you should use (declare) baseUrl variable. Remember the DRY principle.